provider/messaging/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "{{ project-name }}"
+version = "0.1.0"
+edition = "2021"
+resolver = "2"
+
+[dependencies]
+async-trait = "0.1"
+futures = "0.3"
+serde = {version = "1.0", features = ["derive"] }
+tracing = "0.1.37"
+wasmbus-rpc = "0.10.0"
```

Comment:
Should we enable `otel` on our templates?

Reply:
Yeah great idea